### ISSUE TITLE
User.dat File Has Users Already Stored Upon Downloading the Project

### ISSUE THREAD

**Issue by timwalker25** [QihanLu/CS419#8]
There were two users already present in the program when I had downloaded it. This can create some confusion if someone wishes to make users with those names.

**Comment by Kagesss** [comment]
Yes we do have some test accounts exits in the .dat file. However you can delete those account after logging in as admin. It is true that it may cause some confusion. but technically we won't consider it as a bug since no security problem or program crashes will be caused by that.

**Comment by symxmyz233** [comment]
I agree, it is **not a bug.**
